lib/routes/ln_address/ln_address_fee_message.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:c_breez/bloc/lsp/lsp_bloc.dart';
+import 'package:c_breez/bloc/payment_options/payment_options_bloc.dart';
+import 'package:c_breez/widgets/warning_box.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LnAddressFeeMessage extends StatelessWidget {
+  const LnAddressFeeMessage();
+
+  @override
+  Widget build(BuildContext context) {
+    final themeData = Theme.of(context);
+    final lspState = context.watch<LSPBloc>().state;
+    final isChannelOpeningAvailable = lspState?.isChannelOpeningAvailable ?? false;
+    final openingFeeParams = lspState?.lspInfo?.openingFeeParamsList.values.first;
+
+    return isChannelOpeningAvailable
+        ? WarningBox(
+            boxPadding: const EdgeInsets.fromLTRB(16, 30, 16, 16),
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Text(
+                  _formatFeeMessage(context, openingFeeParams!),
+                  style: themeData.textTheme.titleLarge,
+                  textAlign: TextAlign.center,
+                ),
+              ],
+            ),
+          )
+        : const SizedBox();
+  }
+
+  String _formatFeeMessage(BuildContext context, OpeningFeeParams openingFeeParams) {
+    final texts = context.texts();
+    final currencyState = context.read<CurrencyBloc>().state;
+    final accountState = context.read<AccountBloc>().state;
+    final paymentOptionsState = context.read<PaymentOptionsBloc>().state;
+    // Proportional value is ppm(parts per million)
+    final proportionalPercent = (openingFeeParams.proportional / (1000000 / 100)).toString();
+    final liquiditySats = accountState.maxInboundLiquidity;
+    final liquidityFormatted = currencyState.bitcoinCurrency.format(liquiditySats);
+    final liquidityAboveMin = liquiditySats > 1;
+
+    // Calculate the maximum receivable amount within the fee limit (in millisatoshis)
+    final channelFeeLimitSat = paymentOptionsState.channelFeeLimitMsat ~/ 1000;
+    // Calculate the maximum sendable amount (in sats)
+    final maxSendableSats = maxReceivableSat(accountState, channelFeeLimitSat, openingFeeParams);
+    final maxSendableFormatted = currencyState.bitcoinCurrency.format(maxSendableSats);
+
+    // Get the minimum sendable amount (in sats), can not be less than 1 or more than maxSendable
+    final minSendableSats = (liquidityAboveMin) ? 1 : openingFeeParams.minMsat ~/ 1000;
+    final minSendableAboveMin = minSendableSats > 1;
+    final minSendableFormatted = currencyState.bitcoinCurrency.format(minSendableSats);
+    if (!minSendableAboveMin) {
+      return "Minimum sendable amount can't be less than ${currencyState.bitcoinCurrency.format(1)}.";
+    }
+    if (minSendableSats > maxSendableSats) {
+      return "Minimum sendable amount can't be greater than maximum sendable amount.";
+    }
+
+    final isFeeApplicable = maxSendableSats < liquiditySats;
```

Comment:
Fee can be applicable if the max sendable is more than the liquidity
```suggestion
    final isFeeApplicable = maxSendableSats > liquiditySats;
```

Reply:
Done [Address feedback](https://github.com/breez/c-breez/pull/842/commits/ef57956fdd2efeae4e955b09a63f212fb8903870)